lib/src/rules/empty_statements.dart

```diff
@@ -71,6 +71,7 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitEmptyStatement(EmptyStatement node) {
+    if (node.parent is SwitchPatternCase) return;
```

Comment:
I wonder whether we want to only allow empty statements when the case contains only empty statements and it's the last such statement. For example, consider the following cases
```
case a:
  print('a');
  ; // [LINT]
case b:
  ; // [LINT]
  print('b');
case c:
  ; // [LINT]
  ; // [OK]
case d:
  ; // [OK]
```

Reply:
Ah, nice. 💯

Reply:
I updated the implementation and tests. PTAL!